### PR TITLE
feat: add wallet type documentation

### PR DESCRIPTION
**Pull request by ChenRen93**
https://www.pivotaltracker.com/story/show/179201536

Added documentation for wallet type integration

### CODE REVIEW

File: docs/document-creator/config-file.md [hunk 1]
```diff
@@ -46,8 +67,14 @@ As of now, we only cater to 3 networks.
 ---
 
 ## Wallet field
+The `"wallet"` field is a string that refers to your ethereum wallet. The configuration file support two type of wallet option.
+
```
Comment (simboonlong) [inline docs/document-creator/config-file.md:71]:
```suggestion
The `"wallet"` field is a string that refers to your ethereum wallet. The configuration file supports two types of wallet option.

```

File: docs/document-creator/config-file.md [hunk 2]
```diff
@@ -46,8 +67,14 @@ As of now, we only cater to 3 networks.
...
+- ENCRYPTED_JSON
+- AWS_KMS
 
-The `"wallet"` field is a string that refers to your ethereum wallet. You can derive it from either one of these methods:
+#### ENCRYPTED_JSON Wallet
+
+The wallet field can be derive it from either one of these methods:
```
Comment (simboonlong) [inline docs/document-creator/config-file.md:77]:
```suggestion

The wallet field can be derived from either one of these methods:
```

File: docs/document-creator/config-file.md [hunk 3]
```diff
@@ -97,7 +124,18 @@ wallet.encrypt("<Enter a password of your choice here>").then((str) => console.l
...
+Ethereum uses Elliptic Curve Digital Signing Algorithm (ECDSA). More specifically, the elliptic curve being used for transaction signing is secp256k1
+
+1) If you don't have a Aws Kms Wallet, you can refer to [documentation](https://luhenning.medium.com/the-dark-side-of-the-elliptic-curve-signing-ethereum-transactions-with-aws-kms-in-javascript-83610d9a6f81) to create a ECC_SECG_P256K1 Key.
+
+2) Connect the Aws Kms Key to an Aws Iam User and enable the signing access control.
+
+3) The IAM credential and KMS Key ID can now be entered into the `"wallet"` field in the [config file](#aws-kms-wallet).
+
 
 ---
```
Comment (simboonlong) [inline docs/document-creator/config-file.md:140]: maybe can include a link to `https://docs.tradetrust.io/docs/advanced/aws-kms/overview`

**Review by simboonlong — APPROVED** [review]
lgtm

File: docs/document-creator/config-file.md (hunk 3 above)
Comment (ChenRen93) [reply docs/document-creator/config-file.md:140]: i update the documentation link to this overview instead of the external guide link

**Pull request merged**